.scaffold/assets/update-assets.php

```
@@ -47,53 +92,188 @@ function main(): void {
+ *   Path to this script.
+ * @param string $name
+ *   The job name.
+ * @param string $workspace_dir
+ *   Path to the workspace directory.
+ * @param string $cwd
+ *   Working directory for the process.
+ *
+ * @return array{exit_code: int, output: string}
+ *   The exit code and combined output.
+ */
+function runWorker(string $script_path, string $name, string $workspace_dir, string $cwd): array {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$cwd'. (undefined)

(UnusedFormalParameter)